interface/src/app/analyze/analysis/analysis.js

```diff
@@ -57,6 +57,10 @@ AnnotationType, SampleBin) {
     });
   };
 
+  $scope.showVolcanoPlot = function() {
+    $state.go('volcano');
```

Comment:
Yes, we need parameters but it's not implemented yet. Two reasons: for (1), see the comment I made above in response to Dongbo in `view.js`, line 25, about the selection. There are other parameters that would be useful, too. We should be able to specify the sample groups that way, for example. Reason (2) that these parameters aren't implemented yet is that I'll need to do some refactoring to make them work.

I guess that reason (2) sounds a bit cryptic without explaining my thinking a little more. With the current implementation, the sample groups are defined in `SampleBin` and pulled out of there (behind the scenes) by `view.js`. If this page were to respond automatically to what's in the URL parameters, I'd need to re-implement a bunch of the code that currently retrieves activity data, *etc.*, inside `SampleBin`. The right way to make this happen is to refactor `SampleBin` (which I plan to do, as I mentioned above) and then use those refactored components to implement the parameters here.

The picture for what components we need and how they fit together has become more clear to me as I see how the volcano plot and heatmap play off of each other. We can evolve the code to get there, but not before our release deadline! This gets noted under issue #76.